<?xml version="1.0"?>

<launch>

	 <!-- "RunViveTracker.sh" -->
         <!--node name="Vrui_server" pkg="vrui_mdf" type="Vrui_server"/-->

	 <!-- "Gazebo" -->

  	 <!-- Start Gazebo with wg world running in (max) realtime -->

  	 <!-- Spawn simulated quadrotor uav -->

  <arg name="world" default="worlds/empty.world"/>
  <arg name="gui" default="true"/>
  <arg name="args" default=""/>
  <arg name="gravity" default="0"/>
  
  <param name="/use_sim_time" value="true" />

  <node name="gazebo" pkg="gazebo_ros" type="gzserver" args="$(arg world) $(arg args)" respawn="false" output="screen"/>
  
	 <!-- spawn model for robonaut -->
	 <node name="spawn_model_robonaut" pkg="gazebo_ros" type="spawn_model"
	 args="-file $(find openvr_headset_ros)/models/robonaut/model.sdf -sdf -x 1 -model r2"/>
	  
	<group if="$(arg gui)">
    <node name="gazebo_gui" pkg="gazebo_ros" type="gzclient" respawn="false" output="screen"/>
  </group>

</launch>

